spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/Engine.scala

```diff
@@ -35,6 +35,7 @@ sealed trait EngineType
 
 case object MklBlas extends EngineType
 
+case object LocalEngine extends EngineType
```

Comment:
what's this? Engine type is the native libraries like mklblas and mkldnn...

Reply:
As mentioned above, a EngineType other than _MklBlas_ has been built to make _modelPoolSize_ == _nCores_.